src/config.ts

```diff
@@ -170,7 +170,10 @@ const defaultConfig: IConfig = {
 };
 
 export function read(): IConfig {
-    const content = fs.readFileSync(`./config/${process.env.NODE_ENV || 'default'}.yaml`, "utf8");
+    const config_dir = process.env.NODE_CONFIG_DIR || "./config";
+    const config_file = `${process.env.NODE_ENV || "default"}.yaml`
+
+    const content = fs.readFileSync(`${config_dir}/${config_file}`, "utf8");
```

Comment:
Should we use [path](https://nodejs.org/api/path.html) parse and join just to be on the safe side here?

Reply:
`join` yes, but use `parse` for what?

Reply:
ahh right, i forgot that paths are just strings sorry